### PR TITLE
Feature/categories

### PR DESCRIPTION
**Pull request by khaledsaadeh**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -16,29 +17,36 @@ const SEARCH_API =
...
       <div className="App">
-      <NavBar>
-        <Search
-          onChangeQuery={setSearch}
-          query={search}
-          onSearch={fetchSearch}
-        />
-      </NavBar>
+        <NavBar>
+          <Route path="/" exact>
```
Comment (hazemturki) [inline src/App.js:40]: Why did you add this? Are we only showing the navabar in the "/" route?

File: src/components/NavBar.js [hunk 2]
```diff
@@ -1,8 +1,15 @@
+import { Link } from "react-router-dom";
```
Comment (hazemturki) [inline src/components/NavBar.js:1]: This file isn't formatted

File: src/App.js (hunk 1 above)
Comment (khaledsaadeh) [reply src/App.js:40]: No, I'm only showing the search bar and the category selection in the "/" route,

File: src/components/Search.js [hunk 3]
```diff
@@ -12,8 +13,9 @@ const Search = (props) => {
         value={props.query}
         onChange={setSearchHandler}
       />
+
       <button
-        className="bg-blue-600 hover:bg-blue-700 text-white font-bold py-2 px-4 rounded"
+        className="items-center p-2 rounded-md text-gray-400 hover:text-white hover:bg-gray-700 focus:outline-none focus:ring-2 focus:ring-white"
```
Comment (hazemturki) [inline src/components/Search.js:18]: items-center is doing nothing here.
items-center is a class that is special to displays of type flex.
The button display is is block, therefor items-center is useless here

**Review by hazemturki — APPROVED** [review]
